ci_scripts/tf-vars.sh

```diff
@@ -40,9 +40,6 @@ elif [ "${OS}" = "${TC_MSYS_VERSION}" ]; then
     export TASKCLUSTER_ARTIFACTS="$(cygpath ${TASKCLUSTER_ARTIFACTS})"
 
     export DS_ROOT_TASK=${TASKCLUSTER_TASK_DIR}
-    export BAZEL_VC='C:\Program Files (x86)\Microsoft Visual Studio\2019\BuildTools\VC'
```

Comment:
Don't we run a risk there of running silently with the wrong VC/VS version and having the tensorlow cache not doing it's job in the future, silently (just being super slow)?